style.css

```diff
@@ -0,0 +1,213 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* intro style */
+.container-fluid {
+  position: relative;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #fff;
+  border-bottom-left-radius: 60px;
+  box-shadow: 1px 0 1px #fff;
+}
+
+/* logo style */
+.d-flex1 {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 3rem 0 0;
+}
+
+/* main content style */
+.main {
+  width: 100%;
+  padding: 1rem;
+  background-image: url("./images/header-shapes-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.d-flex2 {
+  display: flex;
+  flex-direction: column;
+}
+
+/* intro style */
+.intro {
+  margin-top: 3.5rem;
+}
+
+h1 {
+  font-size: 3rem;
+  padding-bottom: 1rem;
+}
+
+.intro p {
+  line-height: 2rem;
+}
+
+/* lets connect */
+.connect {
+  color: #7f8cff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 1.2rem;
+  text-transform: uppercase;
+  padding: 1rem 0;
+}
+
+.social-icons {
+  display: flex;
+}
+
+.social-icons a {
+  padding: 0 1.5rem 0 0;
+}
+
+/* end of intro style */
+
+/* article styling */
+article {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.section2 {
```

Comment:
- [ ] Please use CSS Flexbox to align content inside each project. You can add these two CSS rules.
```css
section2{ 
  display: flex;
  flex-direction:column;
}
```
Here are some reasons for using Flexbox
[ Why use FlexBox](https://www.toptal.com/front-end/how-to-build-css-only-smart-layouts-with-flexbox)